Bio/SeqIO/PdbIO.py

```diff
@@ -458,6 +460,8 @@ def CifAtomIterator(handle):
 
     buffer.seek(0)
     mmcif_dict = MMCIF2Dict(buffer)
+    if mmcif_dict == {}:
```

Comment:
Can you check if the buffer object is empty directly? You could then do this two lines up, before ever calling ``MMCIF2Dict``.

Reply:
Done.